.github/workflows/android-custom-build.yml

```diff
@@ -1,15 +1,15 @@
 name: ID PASS - MOSIP Resident Application Custom build
```

Comment:
Also rename this to 
Android Custom Build

Reply:
changed